torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -684,23 +684,29 @@ llvm::Error PyTorchModelLoader::loadLinear(const torch::jit::Node *ptNode) {
   auto outputs = ptNode->outputs();
   assert(inputs.size() == 3);
   assert(outputs.size() == 1);
-
   glow::NodeValue input;
   ASSIGN_VALUE_OR_RETURN_ERR(input,
                              getGlowNodeValue(inputs[LinearInputs::input]));
   glow::NodeValue weights;
   ASSIGN_VALUE_OR_RETURN_ERR(weights,
                              getGlowNodeValue(inputs[LinearInputs::weights]));
 
-  auto *mul = F_.createMatMul("linear_mul", input, weights);
+  // Currently in Pytorch => Glow, we only translate t()+addmm(), t()+mm() and
```

Comment:
This is some pretty baked-in logic, we'll have to revisit this later probably.

Reply:
Exactly, that's why I left some comments here...